### DISCUSSION TITLE
i don't hear nothing

### DISCUSSION THREAD

**Discussion by miky2184** [meltaxa/jukebox-portal#10] [Q&A] [answered]
Hi...great project...i have follow the instruction....but i don't hear nothing.

this is the log:
[2021-02-23 10:13:47,386] INFO: Lego Dimensions base activated.
[2021-02-23 10:13:47,515] INFO: Lightshow is True
[2021-02-23 10:14:03,012] INFO: Spotify activated.
[2021-02-23 10:16:26,368] INFO: Lightshow started for 60.0 seconds.
[2021-02-23 10:17:51,447] INFO: Lightshow started for 60.0 seconds.
[2021-02-23 10:19:20,781] INFO: Lightshow started for 60.0 seconds.

which link have u used to play music?

thanks

**Comment by meltaxa** [comment, marked as answer]
There are several ways to listen to the music via Spotify, all involving using a Spotify device to listen to the selected track. For example, use a Spotify app (say on your phone) to choose which Spotify device to hear the music on: the phone, Raspberry Pi (via raspotify), or a Spotify enabled smart speaker.
* To install [raspotify](https://github.com/dtcooper/raspotify) which will enable the Raspberry Pi as a Spotify device: 
```
curl -sL https://dtcooper.github.io/raspotify/install.sh | sh
```
* In one of the videos, I use my phone as the Spotify device but then pair it to a bluetooth music system.

Let me know how you go. I'll update the documentation for others to follow too.

**Reply by miky2184** [reply]
yes yes thanks... i've done... spotify not found  a default music player...i solved

thanks so much...Is great....your project is great
